src/legacy/core_plugins/vis_type_vislib/public/vislib/helpers/point_series/_get_series.ts

```diff
@@ -69,32 +71,16 @@ export function getSeries(table, chart) {
 
         point.seriesId = seriesId;
         addToSiri(
-          series,
+          seriesMap,
           point,
-          seriesId,
+          seriesId as string,
           seriesLabel,
           y.format,
           zAspect && zAspect.format,
           zAspect && zAspect.title
         );
       });
-    }, new Map())
-    .thru(series => [...series.values()])
-    .value();
-
-  if (multiY) {
-    series = _.sortBy(series, function(siri) {
-      const firstVal = siri.values[0];
-      let y;
-
-      if (firstVal) {
-        y = _.find(aspects.y, function(y) {
-          return y.accessor === firstVal.accessor;
```

Comment:
Here `firstVal` is an object with type `Point`, which doesn't have `accessor` property. It means that `y` will be `undefined` and an iteratee function will always return `series.length` => there's no sense in `sortBy`. That's why I removed this code.